### PR TITLE
Phi, x, and r for xtab

### PR DESCRIPTION
**Pull request by mattansb**

### CODE REVIEW

File: tests/testthat/test-printing.R [hunk 1]
```diff
@@ -94,37 +94,37 @@ if (require("testthat") && require("effectsize")) {
     expect_output(print(e4), regexp = "(Type III)", fixed = TRUE)
   })
 
-  test_that("print | effectsize_std_params", {
```
Comment (mattansb) [inline tests/testthat/test-printing.R:97]: @strengejacke Note these tests should also be in `parameters`

**Comment by mattansb** [comment]
@strengejacke this might break `model_parameters.htest()` for chisq tests.

@IndrajeetPatil some breaking changes here. See `NEW.md`

**Comment by IndrajeetPatil** [comment]
Thanks, will make the necessary changes.

You can note in the CRAN comments that the maintainer has already been informed.

**Comment by mattansb** [comment]
@IndrajeetPatil revdep actually found no issues 😅

**Pull request merged**